nomad-gcp/terraform.tf

```diff
@@ -0,0 +1,8 @@
+provider "google" {
```

Comment:
Accidental file add?

Reply:
The terraform scripts won't run without the google provider modules being added. I'm not sure if it's due to the recent update to using terraform v0.15.4 or something else. I can remove this file, but it will be needed by the integration workflow to avoid errors.

Reply:
Errors seen without `terraform.tf`:
![image](https://user-images.githubusercontent.com/38337779/129629634-7a979eab-0e13-4218-8307-d7f4904f9cfb.png)

Reply:
Right, that's where they were being created before in the dedicated clients: https://github.com/circleci/server/commit/c45aff4067b0b4b57a6b03e88b12fa7fc3e9faeb#diff-25d9e530131277f792e90853c7d0aee2ab9655fd4b146aba9615d3801338b3dcL264-L274

Since this is a public repo, I would remove that and generate that file via the scheduled refresh job

Reply:
ok! I'll remove this!